src/concepts/analysis_intro.md

```diff
@@ -1,88 +0,0 @@
-# Get Started with Firefox Desktop Data
```

Comment:
This document walked a really confusing line between being a tutorial and explanation. As such I've split it up into two other documents:

* The (IMO) important concepts/explanation parts found their way into "What Data does Mozilla Collect?". Some of this document re-explains things that have a more authoritative explanation elsewhere (e.g. Guiding Principles for Data Infrastructure) and were deleted.
* The tutorial parts were moved to a BigQuery tutorial called "Accessing Desktop Data"